contracts/external/traders/Expiry.sol

```diff
@@ -154,17 +193,12 @@ contract Expiry is
         public
         onlySolo(msg.sender)
     {
-        (
-            uint256 marketId,
-            uint32 expiryTime
-        ) = parseCallArgs(data);
-
-        // don't set expiry time for accounts with positive balance
-        if (expiryTime != 0 && !SOLO_MARGIN.getAccountPar(account, marketId).isNegative()) {
-            return;
+        CallFunctionType callType = abi.decode(data, (CallFunctionType));
```

Comment:
What does `abi.decode` do if the first byte(s) of `data` are not a `CallFunctionType`? (they're > 1) 

Do we need to manually check for that?

Reply:
It hits an assertion error in solidity